parm/archive/gdas.yaml.j2

```diff
@@ -58,7 +58,8 @@ gdas:
 
         # Analysis state
             {% if DO_JEDIATMVAR %}
-        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atmvar.yaml"
+        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atmanlvar.yaml"
+        - "{{ COMIN_ATMOS_ANALYSIS | relpath(ROTDIR) }}/{{ head }}atmanlfv3inc.yaml"	
```

Comment:
Remove white space at the end of this line.  See [this comment](https://github.com/NOAA-EMC/global-workflow/pull/2805#issuecomment-2322855533) in PR #2805